terraform/create_state_bucket/s3.tf

```
@@ -0,0 +1,25 @@
+resource "aws_kms_key" "state_key" {
+  description = "Used by Terraform to store remote state for ashley"
+}
+
+resource "aws_s3_bucket" "state_bucket" {
+  bucket = "ashley-terraform${var.unique_suffix}"
```

I suggest you to use [bucket_prefix](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/s3_bucket#bucket_prefix) instead

```suggestion
  bucket_prefix = "ashley-terraform"
```